ts/Stock/Indicators/LinearRegressionAngle/LinearRegressionAngle.ts

```diff
@@ -154,6 +154,7 @@ export default LinearRegressionAngleIndicator;
  * @product   highstock
  * @excluding dataParser,dataURL
  * @requires  stock/indicators/indicators
+ * @requires  stock/indicators/linearregression
```

Comment:
Just checked the dist folder: there is no `linearregression`, `linearregressionangle` etc modules. Only one, that combines them all: https://code.highcharts.com/stock/indicators/regressions.src.js